.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,68 @@
+# Contributing
+## Found a Bug?
+If you find a bug in the source code, you can help us by submitting an issue to our GitHub Repository. Even better, you can submit a Pull Request with a fix.
```

Comment:
Do we make the same? 

On the iOS side, there is even a tutorial link:
`**Considering making your first Pull Request?** You can learn how from this *free* series [How to Contribute to an Open Source Project on GitHub](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github)`

Reply:
Sure, we can add this. I just copied these from Remote Config, so there are minor differences between the two.